examples/adaptive_library/BPMTriangularRotorNotches.py

```diff
@@ -50,6 +50,8 @@
 # Import ``pymotorcad`` to access Motor-CAD. Import ``triangular_notch`` to create the notch
```

Comment:
```suggestion
# Import the ``pymotorcad`` package to access Motor-CAD. Import the ``triangular_notch`` package to create the notch
```
All code entities should be followed by a noun indicating the type of object it is.